hles/extraction/src/main/scala/org/broadinstitute/monster/dap/Args.scala

```diff
@@ -10,7 +10,7 @@ import org.broadinstitute.monster.buildinfo.DogAgingHlesExtractionBuildInfo
 
 @AppName("DAP HLE extraction pipeline")
 @AppVersion(DogAgingHlesExtractionBuildInfo.version)
-@ProgName("org.broadinstitute.monster.dap.HLESurveyExtractionPipeline")
+@ProgName("dog-aging-ingest")
```

Comment:
❓ I am unable to find any real documentation behind this annotation other than [this example](https://github.com/alexarchambault/case-app#customizing-items-of-the-help--usage-message) in the caseapp readme. It seems like it should only affect the CLI args output, so I am making this change to be more reflective of the actual program name.